src/views/integrations/event-tickets/emails/template-parts/body/event/title.php

```diff
@@ -28,10 +28,12 @@
 <tr>
 	<td class="tec-tickets__email-table-content-event-title-container">
 		<h3 class="tec-tickets__email-table-content-event-title">
-			<?php
-			// phpcs:ignore
-			echo $event->title;
-			?>
+			<a href="<?php echo esc_url( $event->permalink ); ?>" target="_blank">
```

Comment:
if we're using `target="_blank"` we also need to accompany that with `rel="noopener noreferrer"`.